### ISSUE TITLE
exception: smallestImage is undefined (imageSize is undefined)

### ISSUE THREAD

**Issue by 2SC1815J** [mejackreed/Leaflet-IIIF#73]
If options.tileSize is greater than image size, you will have an exception as follows.

https://iiif.dl.itc.u-tokyo.ac.jp/omekac/iiif/6134/manifest
https://iiif.dl.itc.u-tokyo.ac.jp/omekac/iiif-img/1631/info.json

[Leaflet 1.4.0＋Leaflet-IIIF 2.0.1](https://gist.githack.com/2SC1815J/1f636969561f9a7815d4622d9b65a119/raw/1980e94ba2bb4d81c13c433c0b47b32a9a176984/iiif-example2.html)
```
jQuery.Deferred exception: smallestImage is undefined onAdd/<@https://rawgit.com/mejackreed/Leaflet-IIIF/v2.0.1/leaflet-iiif.js:81:1
```

[Leaflet 0.7.7＋Leaflet-IIIF 0.2.0](https://gist.githack.com/2SC1815J/1f636969561f9a7815d4622d9b65a119/raw/1980e94ba2bb4d81c13c433c0b47b32a9a176984/iiif-example1.html)
```
jQuery.Deferred exception: imageSize is undefined _fitBounds@https://rawgit.com/mejackreed/Leaflet-IIIF/v0.2.0/leaflet-iiif.js:94:9
```
(In Leaflet-IIIF 0.2.0, this will happen only when info.json lacks `tiles` property. This property is optional by [spec](https://iiif.io/api/image/2.1/#technical-properties). rel. https://github.com/mejackreed/Leaflet-IIIF/pull/56)

This seems to be caused by [the following points](https://github.com/mejackreed/Leaflet-IIIF/blob/master/leaflet-iiif.js#L204), _this.maxNativeZoom becomes -1.
```
_this.maxNativeZoom = Math.max(ceilLog2(_this.x / _this.options.tileSize),
          ceilLog2(_this.y / _this.options.tileSize));
```

Because of https://github.com/mejackreed/Leaflet-IIIF/issues/64, I still have to use Leaflet-IIIF 0.2.0. The following workaround will be OK?
```
_this.maxNativeZoom = Math.max(ceilLog2(_this.x / _this.options.tileSize),
          ceilLog2(_this.y / _this.options.tileSize), 
          0);
```

**Comment by mejackreed** [comment]
That seems like a good solution. It seems like this is only happening when an explicit `tileSize` option is passed in.

Also what about this (I'm trying to think about future comprehension and readability)

```diff
// Calculates maximum native zoom for the layer
- _this.maxNativeZoom = Math.max(ceilLog2(_this.x / _this.options.tileSize),
-          ceilLog2(_this.y / _this.options.tileSize));

+ _this.maxNativeZoom = Math.max(
+  ceilLog2(_this.x / _this.options.tileSize),
+  ceilLog2(_this.y / _this.options.tileSize),
+  0
+ );
```

**Comment by 2SC1815J** [comment]
Thank you very much! That will be better.
Should I make a pull request to the branch `release-0.2.x` ?

**Comment by mejackreed** [comment]
Resolved in #74
